java/code/src/com/redhat/rhn/taskomatic/task/MinionActionChainExecutor.java

```diff
@@ -35,8 +33,6 @@
  */
 public class MinionActionChainExecutor extends RhnJavaJob {
 
-    private static final Logger LOG = LogManager.getLogger(MinionActionChainExecutor.class);
-
```

Comment:
Weren't your goal to make loggers private? Shouldn't instead be RhnJavaJob logger made private? And keep this here? Or is this because RhnJavaJob is abstract class?

Reply:
That has been my goal at the beginning, but `RhnJavaJob` is the base of a lot of classes... I started doing it and changed my mind in the course of it. Not sure what you guys think about it... Since this not even a report in SonarQube, I left that one as is

Reply:
Since `RhnJavaJob` is an abstract class, I don't as a problem on having the logging defined in there as protected.